File: PgAce.App/Forms/ResultPane.cs
using System.Windows.Forms;
using WeifenLuo.WinFormsUI.Docking;

namespace PgAce.App.Forms
{
    public class ResultPane : DockContent
    {
        public ResultPane()
        {
            Text = "Result";
            var grid = new DataGridView
            {
                Dock = DockStyle.Fill,
            };
            Controls.Add(grid);
        }
    }
}

File: PgAce.App/Forms/ResultPane.vb
Imports System.Windows.Forms
Imports WeifenLuo.WinFormsUI.Docking

Namespace PgAce.App.Forms
    Public Class ResultPane
        Inherits DockContent

        Public Sub New()
            Text = "Result"
            Dim grid = New DataGridView() With {
                .Dock = DockStyle.Fill
            }
            Controls.Add(grid)
        End Sub
    End Class
End Namespace
